src/meshes/t8code_mesh.jl

```diff
@@ -7,8 +7,6 @@ to manage trees and mesh refinement.
 """
 mutable struct T8codeMesh{NDIMS, RealT <: Real, IsParallel, NDIMSP2, NNODES} <:
                AbstractMesh{NDIMS}
-    cmesh       :: Ptr{t8_cmesh} # cpointer to coarse mesh
-    scheme      :: Ptr{t8_eclass_scheme} # cpointer to element scheme
     forest      :: Ptr{t8_forest} # cpointer to forest
```

Comment:
A general remark regarding the `T8codeMesh`: In the `P4estMesh`, we use [`PointerWrapper`s from P4est.jl](https://github.com/trixi-framework/P4est.jl/blob/21de33062c697e9d0ae4c7cf290390440abf47c1/src/pointerwrappers.jl) (see https://github.com/trixi-framework/Trixi.jl/pull/1434) to make some parts of the code more readable. I am not sure if something similar should be done for the `T8codeMesh` to improve readability (and consistency), but this could be something to discuss.

Reply:
This is indeed a good advice and I will keep that in mind for future usage. However, in t8code there are always setter and getter routines. Directy access to member fields of the `t8_forest` structure is not necessary and dissuaded.